cas-server-core-audit/src/main/java/org/jasig/cas/audit/spi/TicketOrCredentialPrincipalResolver.java

```diff
@@ -98,13 +102,13 @@ private String resolveArgument(final Object arg1) {
         } else if (arg1 instanceof String) {
             try {
                 final Ticket ticket = this.centralAuthenticationService.getTicket((String) arg1, Ticket.class);
+                org.jasig.cas.authentication.Authentication casAuthentication = null;
```

Comment:
Sorry @leleuj . That Spring Security `Authentication` type was/is in the `4.1.2`. Will do the propery import. Sorry for the confusion.